index.html

```diff
@@ -0,0 +1,176 @@
+        </div>
+      </section>
+      <section class="program" id="program">
+        <h3 class="program-title">Main Program</h3>
+        <div class="program-container">
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/motivation.png" alt="motivation-icone" />
+              <span>Motivation</span>
+            </div>
+            <div class="right">
+              Listen to the motivation form various personalities about the
+              message of sharing and opening opportunities to the less
+              privileged in your community.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/forum.png" alt="forum-icone" />
+              <span>Forum</span>
+            </div>
+            <div class="right">
+              Have the time to share your toughts and opinions with our team for
+              each topic.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/lecture.png" alt="lecture-icon" />
+              <span>Donation</span>
+            </div>
+            <div class="right">
+              The value of life is not in its duration, but in its donation.
+              What a privilege to be here on the planet to contribute your
+              unique donation to humankind!
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/support.png" alt="support-icone" />
+              <span>Support</span>
+            </div>
+            <div class="right">
+              Whatever little it maybe, it means a lot to someone out there.
+              Support the needy in our community.
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="left">
+              <img src="./images/workShop.png" alt="workshop-icone" />
+              <span>Workshop</span>
+            </div>
+            <div class="right">
+              Get oppotunities to network with GC affiliates around the globe.
+            </div>
+          </div>
+        </div>
+        <div class="join-btn">
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+      <section id="sponsor" class="sponsor"></section>
+      <section class="partner">
+        <h3 class="partner-title">Partner</h3>
+        <div class="partner-container">
+          <figure>
+            <img src="./images/mozilla.png" alt="img1" />
+          </figure>
+          <figure>
+            <img src="./images/google.png" alt="img2" />
+          </figure>
+          <figure>
+            <img src="./images/naver.png" alt="img3" />
+          </figure>
+          <figure>
+            <img src="./images/airbnb.jpg" alt="img4" />
+          </figure>
+        </div>
+      </section>
```

Comment:
- [ ] Your partner section on mobile is overflowing horizontally, I think this is as a result of the CSS style you gave your class `partner-container figure` a width of 200% and also `.partner img` width of 150px, kindly use media query to fix this
![image](https://user-images.githubusercontent.com/66097371/164679824-4cd5883f-bdb8-4362-ba0d-79e977b304eb.png)